src/host_capabilities/mod.rs

```diff
@@ -57,14 +80,9 @@ pub enum CallbackRequestType {
     SigstoreGithubActionsVerify {
         /// String pointing to the object (e.g.: `registry.testing.lan/busybox:1.0.0`)
         image: String,
-        /// owner of the repository. E.g: octocat
-        owner: String,
-        /// Optional - Repo of the GH Action workflow that signed the artifact. E.g: example-repo
-        repo: Option<String>,
+        // GitHub Actions information that must be found in the signature
+        github_actions: KeylessGithubActionsInfo,
```

Comment:
Why do we need to move that to a dedicated struct? Having the two fields of this `KeylessGithubActionsInfo` struct embedded into `SigstoreVerificationInputV2::SigstoreGithubActionsVerify` would be better IMHO. That's because there are less risks of accidentally breaking the API we have between the host and the guest.

To be concrete, we might change `KeylessGithubActionsInfo` and then break the v2 api of github verification

Reply:
We moved that to a dedicate struct to have a similar structure to the other verification types, and because there were no [mentions to githubActions](https://github.com/kubewarden/verify-image-signatures/pull/25#discussion_r916696094) 
Maybe we can just rename the original fields `repo` and `owner` to `github_repo` and `github_owner` and remove the dedicated struct?

Reply:
with this struct settings would be like:
``` yaml
signatures:
- image: "ghcr.io/kubewarden/*"
  github_actions:
    owner: "kubewarden"
    repo: "app-example" #optional
```

but we can remove it, then it would be:

``` yaml
signatures:
- image: "ghcr.io/kubewarden/*"
  github_owner: "kubewarden"
  github_repo: "app-example" #optional
```

before it was just:


``` yaml
signatures:
- image: "ghcr.io/kubewarden/*"
  owner: "kubewarden"
  repo: "app-example" #optional
```

Reply:
Well, in the past we didn't have the internal tag from Serde, so it was ambiguous. Now the struct has a `type: "SigstoreGithubActionsVerify"` so it should be ok.

Reply:
Users won't see this internal tag. It might have been a good idea to add a kind attribute to specify which kind of signature it is. Anyway I'm happy to leave it as it was before

Reply:
So, @flavio, we have decided to revert the last commit, removing the `KeylessGithubActionsInfo` struct from the sdk, as it was. And we will add that struct in the policy, to deserialize the settings. That way the policy settings look like:
```yaml
signatures:
- image: "ghcr.io/kubewarden/*"
  github_actions:
    owner: "kubewarden"
    repo: "app-example" #optional
```

Does that sound ok?

Reply:
yes, the policy settings proposed above look good to me :+1:

Reply:
Perfect, dropped the last commit here and in policy-evaluator.